lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb

```diff
@@ -69,13 +70,25 @@ def disconnect_sql
 
     def log(level, msg)
       Thread.current["tracking_label"] = @tracking_label
-      $miq_ae_logger.send(level, "<AEMethod #{current_method}> #{msg}")
+      if @log_prefix.blank?
+        $miq_ae_logger.send(level, "<AEMethod #{current_method}> #{msg}")
+      else
+        $miq_ae_logger.send(level, "<AEMethod #{current_method}> #{@log_prefix} #{msg}")
+      end
     end
 
     def set_state_var(name, value)
       @persist_state_hash[name] = value
     end
 
+    def set_log_prefix(prefix)
+      @log_prefix = prefix
+    end
+
+    def get_log_prefix
```

Comment:
Can we use `log_prefix=(prefix)` and `log_prefix` instead?

Reply:
@gmcculloug 
Since this is for external consumption we have been using gets and sets so far.
$evm.get_state_var
$evm.set_state_var

So along the same lines I have kept it as
$evm.set_log_prefix &
$evm.get_log_prefix